src/main/java/org/apache/sling/api/resource/package-info.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-@Version("2.12.2")
```

Comment:
If we can we should avoid increasing this package version, it breaks the resource resolver.
So maybe we should move this class somewhere else

Reply:
:-/ any suggestion @cziegeler ?

Reply:
best move it to the wrappers package then as well.

Reply:
+1 that's what I was thinking as well

Reply:
+1